wiki/mnmg/Using_Infiniband_for_MNMG.md

```diff
@@ -0,0 +1,388 @@
+# Using Infiniband for Multi-Node Multi-GPU cuML
+
+These instructions outline how to run multi-node multi-GPU cuML on devices with Infiniband. These instructions assume the necessary Infiniband hardware has already been installed and the relevant software has already been configured to enable communication over the Infiniband devices.
+
+The steps in this wiki post have been largely adapted from the [Experiments in High Performance Networking with UCX and DGX](https://blog.dask.org/2019/06/09/ucx-dgx) blog by Matthew Rocklin and Rick Zamora.
+
+## 1. Install UCX 
+
+### From Conda
+
+Note: this package currently requires CUDA9.2. A CUDA10 package is in the works.
+
+`conda install -c conda-forge -c jakirkham/label/ucx cudatoolkit=9.2 ucx-proc=*=gpu ucx python=3.7`
```

Comment:
```suggestion
`conda install -c conda-forge -c jakirkham/label/ucx cudatoolkit=9.2 ucx-proc=*=gpu ucx python=3.7`
```

perhaps add a note that the `jakirkham` is temporary, CUDA10 and the final location are in the works